[liblzma] Build error on x64-windows

Package: liblzma:x64-windows@5.8.0

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.43.34809.0
-    vcpkg-tool version: 2025-03-22-d4dd14112c2026ee57e59f966ecfa3b9137bd143
    vcpkg-scripts version: b328539dd9 2025-04-05 (3 hours ago)

**To Reproduce**

`vcpkg install liblzma`

**Failure logs**

```
-- Using cached tukaani-project-xz-v5-32c65500.8.0.tar.gz
-- Cleaning sources at E:/.vcpkg/buildtrees/liblzma/src/v5.8.0-26427e6c0b.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source E:/.vcpkg/downloads/tukaani-project-xz-v5-32c65500.8.0.tar.gz
-- Applying patch build-tools.patch
-- Using source at E:/.vcpkg/buildtrees/liblzma/src/v5.8.0-26427e6c0b.clean
-- Configuring x64-windows
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: E:/.vcpkg/downloads/tools/cmake-3.30.1-windows/cmake-3.30.1-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j13
    Working Directory: E:/.vcpkg/buildtrees/liblzma/x64-windows-dbg
    See logs for more information:
      E:\.vcpkg\buildtrees\liblzma\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/liblzma/portfile.cmake:37 (vcpkg_cmake_install)
  scripts/ports.cmake:206 (include)



```

<details><summary>E:\.vcpkg\buildtrees\liblzma\install-x64-windows-dbg-out.log</summary>

```
Change Dir: 'E:/.vcpkg/buildtrees/liblzma/x64-windows-dbg'

Run Build Command(s): E:/.vcpkg/downloads/tools/ninja/1.12.1-windows/ninja.exe -v -v -j13 install
[1/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\block_util.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\block_util.c
FAILED: CMakeFiles/liblzma.dir/src/liblzma/common/block_util.c.obj
E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\block_util.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\block_util.c
E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\sysdefs.h(175): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
[2/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\check\check.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check\check.c
FAILED: CMakeFiles/liblzma.dir/src/liblzma/check/check.c.obj
E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\check\check.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check\check.c
E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\sysdefs.h(175): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
[3/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\hardware_physmem.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\hardware_physmem.c
FAILED: CMakeFiles/liblzma.dir/src/liblzma/common/hardware_physmem.c.obj
E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\hardware_physmem.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\hardware_physmem.c
E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\sysdefs.h(175): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
[4/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\common.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\common.c
FAILED: CMakeFiles/liblzma.dir/src/liblzma/common/common.c.obj
E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\common.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\common.c
E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\sysdefs.h(175): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
[5/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\filter_common.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\filter_common.c
FAILED: CMakeFiles/liblzma.dir/src/liblzma/common/filter_common.c.obj
E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\filter_common.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\filter_common.c
E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\sysdefs.h(175): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
[6/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\easy_preset.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\easy_preset.c
FAILED: CMakeFiles/liblzma.dir/src/liblzma/common/easy_preset.c.obj
E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\easy_preset.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\easy_preset.c
E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\sysdefs.h(175): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
[7/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\index.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\index.c
FAILED: CMakeFiles/liblzma.dir/src/liblzma/common/index.c.obj
E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\index.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\index.c
E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\sysdefs.h(175): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
[8/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\vli_size.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\vli_size.c
FAILED: CMakeFiles/liblzma.dir/src/liblzma/common/vli_size.c.obj
E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\vli_size.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\vli_size.c
E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\sysdefs.h(175): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
[9/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\string_conversion.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\string_conversion.c
FAILED: CMakeFiles/liblzma.dir/src/liblzma/common/string_conversion.c.obj
E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\string_conversion.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\string_conversion.c
E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\sysdefs.h(175): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
[10/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\stream_flags_common.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\stream_flags_common.c
FAILED: CMakeFiles/liblzma.dir/src/liblzma/common/stream_flags_common.c.obj
E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\stream_flags_common.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\stream_flags_common.c
E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\sysdefs.h(175): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
[11/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\hardware_cputhreads.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\hardware_cputhreads.c
FAILED: CMakeFiles/liblzma.dir/src/liblzma/common/hardware_cputhreads.c.obj
E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\liblzma\common\hardware_cputhreads.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common\hardware_cputhreads.c
E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\sysdefs.h(175): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
[12/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\common\tuklib_physmem.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\tuklib_physmem.c
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\winbase.h(9531): warning C5105: macro expansion producing 'defined' has undefined behavior
[13/82] E:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   -DDLL_EXPORT -DHAVE_CHECK_CRC32 -DHAVE_CHECK_CRC64 -DHAVE_CHECK_SHA256 -DHAVE_DECODERS -DHAVE_DECODER_ARM -DHAVE_DECODER_ARM64 -DHAVE_DECODER_ARMTHUMB -DHAVE_DECODER_DELTA -DHAVE_DECODER_IA64 -DHAVE_DECODER_LZMA1 -DHAVE_DECODER_LZMA2 -DHAVE_DECODER_POWERPC -DHAVE_DECODER_RISCV -DHAVE_DECODER_SPARC -DHAVE_DECODER_X86 -DHAVE_ENCODERS -DHAVE_ENCODER_ARM -DHAVE_ENCODER_ARM64 -DHAVE_ENCODER_ARMTHUMB -DHAVE_ENCODER_DELTA -DHAVE_ENCODER_IA64 -DHAVE_ENCODER_LZMA1 -DHAVE_ENCODER_LZMA2 -DHAVE_ENCODER_POWERPC -DHAVE_ENCODER_RISCV -DHAVE_ENCODER_SPARC -DHAVE_ENCODER_X86 -DHAVE_IMMINTRIN_H -DHAVE_INTTYPES_H -DHAVE_LZIP_DECODER -DHAVE_MF_BT2 -DHAVE_MF_BT3 -DHAVE_MF_BT4 -DHAVE_MF_HC3 -DHAVE_MF_HC4 -DHAVE_STDBOOL_H -DHAVE_STDINT_H -DHAVE_USABLE_CLMUL -DHAVE_VISIBILITY=0 -DHAVE__BOOL -DHAVE__MM_MOVEMASK_EPI8 -DHAVE___BUILTIN_ASSUME_ALIGNED -DMYTHREAD_VISTA -DPACKAGE_BUGREPORT=\"xz@tukaani.org\" -DPACKAGE_NAME="\"XZ Utils\"" -DPACKAGE_URL=\"https://tukaani.org/xz/\" -DTUKLIB_FAST_UNALIGNED_ACCESS -DTUKLIB_SYMBOL_PREFIX=lzma_ -Dliblzma_EXPORTS -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\api -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\common -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\check -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lz -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\rangecoder -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\lzma -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\delta -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\liblzma\simple -IE:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  -std:c11 -MDd /showIncludes /FoCMakeFiles\liblzma.dir\src\common\tuklib_cpucores.c.obj /FdCMakeFiles\liblzma.dir\ /FS -c E:\.vcpkg\buildtrees\liblzma\src\v5.8.0-26427e6c0b.clean\src\common\tuklib_cpucores.c
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\winbase.h(9531): warning C5105: macro expansion producing 'defined' has undefined behavior
ninja: build stopped: subcommand failed.
```
</details>

--- a/ports/liblzma/portfile.cmake
+++ b/ports/liblzma/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO tukaani-project/xz
     REF "v${VERSION}"
-    SHA512 32c65500ccb49f598d88bca27cdd7bff35b505f16736ed341797eb308dc7fc9f4b01a9c8cacbecd6480701a2f8427777d476504eced663fc4f8b161f0e16adec
+    SHA512 2f51fb316adb2962e0f2ef6ccc8b544cdc45087b9ad26dcd33f2025784be56578ab937c618e5826b2220b49b79b8581dcb8c6d43cd50ded7ad9de9fe61610f46
     HEAD_REF master
     PATCHES
         build-tools.patch

--- a/ports/liblzma/vcpkg.json
+++ b/ports/liblzma/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "liblzma",
-  "version": "5.8.0",
+  "version": "5.8.1",
   "description": "Compression library with an API similar to that of zlib.",
   "homepage": "https://tukaani.org/xz/",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4861,7 +4861,7 @@
       "port-version": 1
     },
     "liblzma": {
-      "baseline": "5.8.0",
+      "baseline": "5.8.1",
       "port-version": 0
     },
     "libmad": {

--- a/versions/l-/liblzma.json
+++ b/versions/l-/liblzma.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "55eb3803f1d0388aebca8fd293963721bbf0092c",
+      "version": "5.8.1",
+      "port-version": 0
+    },
+    {
       "git-tree": "d1fb572d9cdc913cb384d358e5746a669961b004",
       "version": "5.8.0",
       "port-version": 0

